src/main/kotlin/com/back/domain/member/service/MemberService.kt

```
@@ -0,0 +1,256 @@
+            .orElseThrow {
+                ServiceException(
+                    ResultCode.MEMBER_NOT_FOUND.code(),
+                    "회원을 찾을 수 없습니다."
+                )
+            }
+
+        // 기존 프로필 이미지가 있다면 삭제
+        member.profileUrl?.takeIf { it.isNotEmpty() }?.let { oldProfileUrl ->
+            try {
+                fileStorageService.deletePhysicalFile(oldProfileUrl)
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)